store/handler.go

```diff
@@ -145,9 +145,16 @@ func (rh requestHandler) EncodeResponse(ctx context.Context, response http.Respo
 			return nil
 		}
 		if item, ok := value.(OwnableItem); ok {
-			data, err := json.Marshal(&item.Item)
-			if err != nil {
-				return err
+			var (
+				data = []byte(`{}`)
```

Comment:
if  ttl >= 0 a 404 should be returned. not an empty json object

Reply:
Good catch! Made a change assuming you meant ttl <= 0

Reply:
correct